docs/track.rst

```diff
@@ -3056,6 +3056,39 @@ The following meta data is always returned:
 * ``weight``: The number of fetched pages. Should always equal to the ``pages`` parameter.
 * ``unit``: The unit in which to interpret ``weight``. Always "ops".
 
+downsampling
```

Comment:
What about `downsample`? Similarly to how we call our query operation `search` as opposed to `searching`